siphon/catalog.py

```diff
@@ -335,15 +337,14 @@ def make_access_urls(self, catalog_url, all_services, metadata=None):
         self.access_urls = access_urls
 
     def add_access_element_info(self, access_element):
+        """Create an access method from a catalog element."""
```

Comment:
At some point I was in the habit of using raw to make sure LaTeX math renders properly. Not really needed here.